src/index.js

```diff
@@ -2,15 +2,21 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 import { makeServer } from "./server";
-import {BrowserRouter} from "react-router-dom"
+import { BrowserRouter } from "react-router-dom";
+import { LoginContextProvider } from "../src/contexts/login-context"
+import { SignupContextProvider } from "../src/contexts/signup-context"
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
-      <App />
+      <LoginContextProvider>
+        <SignupContextProvider>
+          <App/>
+        </SignupContextProvider>
+      </LoginContextProvider>
```

Comment:
And instead of adding the providers in index.js you can create a new file called as CombinedProviders.jsx

```
import React from "react";
import { LoginContextProvider, SignupContextProvider } from "LOCATION WHERE CONTEXTS ARE STORED";

const CombinedProvider = ({ children }) => {
    return (
       <LoginContextProvider>
        <SignupContextProvider>
                {children}
 </SignupContextProvider>
      </LoginContextProvider>
    )
}

export { CombinedProvider };
```
And then you can start using it in your index.js file just shifting your work from one file to another

Reply:
thanks for the suggestion
will surely add in next pr